config/locales/en.yml

```diff
@@ -88,6 +88,9 @@ en:
       following: "following"
       no_posts: "No posts yet"
       private_account: "This account is private"
+    declined:
+      title: "Black list"
```

Comment:
Please change it to "Block list". In the future use "Block/Allow" instead of "Block/Allow". It's more loyal.